R/methods-epi_df.R

```diff
@@ -404,7 +459,7 @@ arrange_row_canonical.default <- function(x, ...) {
 arrange_row_canonical.epi_df <- function(x, ...) {
   rlang::check_dots_empty()
   cols <- key_colnames(x)
-  x %>% dplyr::arrange(dplyr::across(dplyr::all_of(cols)))
+  x[vctrs::vec_order(x[cols]), ]
```

Comment:
side-note: would be nice to get a "top 10 vctrs tricks" walkthrough from you at some point, might be nice for the rest of us to benefit from your learnings!